text/0008-scoped-gems.md

```diff
@@ -152,16 +202,25 @@ An alternative to this approach would be to reserve prefixes, like `aws-sdk-` fo
 
 In the current specification (although not documented), dots (`.`) are allowed. There are [a few “hidden” gems](https://rubygems.org/search?query=.) that use dots. If adding the `@` character is not desired, the dot character can be re-used to handle scoping, perhaps with the pattern `scope.gem_name` such as `aws-sdk.s3`.
 
+## Alternative 3 - `@scope/gem_name` pattern.
+
+While it might be preferable to use this pattern, it has implementation and legacy complications, specifically with forward slash `/`. Gem install and build will assume `@scope` as a directory and will fail. Gems installed in a location is currently flat and would have impact on existing tooling if installed within folders. Unless the `/` is escaped, the RubyGems.org API will also have issues resolving and displaying scoped gems because of Rails resource routing. This approach can be used with a more thought out design and a willingness for making large and downstream changes.
```

Comment:
I think this should be fleshed out and seriously considered. For me it seems like short term ease of implementation vs long term pain.

Reply:
Other than syntax, what benefit does it bring?